homeassistant/components/dlink/strings.json

```diff
@@ -8,6 +8,13 @@
           "username": "[%key:common::config_flow::data::username%]",
           "use_legacy_protocol": "Use legacy protocol"
         }
+      },
+      "confirm_discovery": {
+        "data": {
+          "password": "Password (default: PIN code on the back)",
+          "username": "[%key:common::config_flow::data::username%]",
+          "use_legacy_protocol": "Use legacy protocol"
```

Comment:
Please reference the `use_legacy_protocol` in the `user` step (example in `homekit` `strings.json`) so we don't have double work for translators.